docs/en/latest/discovery/nacos.md

```diff
@@ -124,7 +124,7 @@ The formatted response as below:
   "node": {
```

Comment:
It would be better if we can add a table of "discovery args" to `nacos.md` like https://github.com/apache/apisix/blob/master/docs/en/latest/plugins/limit-count.md#attributes

Reply:
done.